File: src/main/java/de/steklopod/sec02/Lec01FluxIntro.java
package de.steklopod.sec02;

import de.steklopod.courseutil.Util;
import reactor.core.publisher.Flux;

public class Lec01FluxIntro {

    public static void main(String[] args) {

        Flux<Object> flux = Flux.just(1,2, 3, "a", Util.faker().name().fullName());

        flux.subscribe(
                Util.onNext(),
                Util.onError(),
                Util.onComplete());


    }

}

File: src/main/kotlin/de/steklopod/sec02/Lec01FluxIntro.kt
package de.steklopod.sec02

import de.steklopod.utils.Util.faker
import de.steklopod.utils.Util.onComplete
import de.steklopod.utils.Util.onError
import de.steklopod.utils.Util.onNext
import reactor.core.publisher.Flux

object Lec01FluxIntro {
    @JvmStatic
    fun main(args: Array<String>) {
        val flux = Flux.just<Any>(1, 2, 3, "a", faker().name().fullName())
        flux.subscribe(
            onNext(),
            onError(),
            onComplete()
        )
    }
}
